src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -460,19 +459,27 @@ public function preprocessEditorialLabel($cdbid, $settings)
             $publisher = $article['publisher'];
             $showPublisher = in_array($publisher, $settings['publishers']);
             $publisherInArray = in_array($publisher, $publishers);
-            if (!$publisherInArray && !$limitPublishers) {
-                $publishers[] = $publisher;
-            }
-            if (!$publisherInArray && ($limitPublishers && $showPublisher)) {
+            if (!$limitPublishers || $showPublisher) {
                 $publishers[] = $publisher;
             }
+
+            $publishers = array_unique($publishers);
+
         }
 
         if (empty($publishers)) {
             return null;
         }
 
-        $label = 'UiTip van ' . implode(", ", $publishers);
+        $editorialLabelText = 'UiT-tip van ';
+        $label = $editorialLabelText . implode(", ", $publishers);
+
+        if (count($publishers) > 1 ) {
+          $lastPublisher = array_pop($publisher);
+          $label = $editorialLabelText . implode(", ", $publishers);
+          $label .= ' en ' . $lastPublisher;
+        }
```

Comment:
Try to limit commits to a single change. As soon as you have to use the word "and" (or in your case `+`) in your message, it's usually a sign that you're committing too much at once. You can always stage individual chunks/lines, you don't need to stage all your changes at once. (If you need help with that let me know)

Reply:
Good point! I will pay attention to this in my future commits.